command/agent/template/template.go

```diff
@@ -117,28 +108,26 @@ func (ts *Server) Run(ctx context.Context, incoming chan string, templates []*ct
 						Token: latestToken,
```

Comment:
Small nit, but you could use `ctconfig.String(latestToken)` here and have `latestToken` be a string instead of dealing with pointers directly.

Reply:
I followed the pattern that the [Sink Server uses][1]. As it's a nit, I'm inclined to stay consistent with Sink. 


[1]: https://github.com/hashicorp/vault/blob/f330ca37526e75f5ba5c8d7baf721cd7c5c8b776/command/agent/sink/sink.go#L73-L118